stories/module-analytics-settings.stories.js

```diff
@@ -47,6 +47,12 @@ function filterAnalyticsSettings() {
 	);
 }
 
+const completeModuleData = {
```

Comment:
Why was these needed? If every connected module will need it, we should probably make it a global change.

Reply:
This is only here to decouple at least a little bit from legacy behavior. All our Storybook components currently heavily rely on legacy `global.googlesitekit` data which is terrible to maintain and regenerate (see all the files in `.storybook/data`). This here is used for rely on that data for the most part, but then enforce certain module flags to ensure it is considered active and set up. In the (probably very) long term, we will not have the `global.googlesitekit` legacy data anymore, and this would all be scaffolded individually or use a fixture, a mock creator or something like that.